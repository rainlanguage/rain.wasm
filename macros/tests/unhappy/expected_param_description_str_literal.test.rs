#[macro_use]
extern crate wasm_bindgen_utils_macros;

#[wasm_export]
impl TestStruct {
    #[wasm_export]
    pub fn test_method(
        #[wasm_export(param_description = something)]
        arg: String,
    ) -> Result<String, Error> {
        Ok(arg)
    }
}

struct TestStruct;

fn main() {}