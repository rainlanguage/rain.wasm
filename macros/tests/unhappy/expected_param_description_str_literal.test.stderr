error: expected string literal
 --> tests/unhappy/expected_param_description_str_literal.test.rs:8:23
  |
8 |         #[wasm_export(param_description = something)]
  |                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
